resources/migrations/000_migrations.yaml

```
@@ -16096,6 +16096,59 @@ databaseChangeLog:
               column:
                 name: action_id
 
+  - changeSet:
+      id: v48.00-009
+      author: calherries
+      comment: Added 0.48.0 - Create table_privileges table
+      changes:
+        - createTable:
+            tableName: table_privileges
```

```suggestion
            tableName: table_privileges
```

any reason why we use plural here?

---

My thinking was that each row contains a set of privileges for a table and user role, e.g. select, insert, update, delete. You can't call a single toucan instance a "table privilege", so the table must be "table privileges".